tests/rake_test.py

```diff
@@ -53,13 +53,15 @@ def test_generate_phrases(self):
                        ['define'], ['sequence'], ['one'], ['words'],
                        ['provide'], ['compact', 'representation'], ['document'],
                        ['content']]
+        phrase_list = set(tuple(phrase) for phrase in phrase_list)
```

Comment:
As the expected set is a hard coded list of phrases, maybe its a good idea to create the set in line 52 as a set of tuples rather than taking another pass at it here. Same goes for the other two changes. If you choose not to do it, I will do it myself.